README.MD

```diff
@@ -0,0 +1,114 @@
+### @password-validator/core
+
+<div style="display:flex; gap:10px; flex-wrap:wrap">
+  <a href="https://www.npmjs.com/package/@password-validator/core" style="text-decoration:none">
+    <img src="https://img.shields.io/npm/v/@password-validator/core.svg"  />
+  </a>
+
+  <a href="https://www.npmjs.com/package/@password-validator/core" style="text-decoration:none">
+    <img src="https://img.shields.io/npm/dt/@password-validator/core.svg" />
+  </a>
+
+  <a href="https://github.com/Theshedman/password-validator.js" style="text-decoration:none">
+    <img src="https://img.shields.io/github/workflow/status/Theshedman/password-validator/build-ci?logo=github" />
+  </a>
+
+  <a href="https://github.com/Theshedman/password-validator.js" style="text-decoration:none">
+    <img src="https://img.shields.io/github/languages/top/Theshedman/password-validator.js.svg"  />
+  </a>
+</div>
+
+---
+
+
+A zero dependency library for password validation that lets you compose and combine multiple rules to create a validator that suits your needs.
+
+### Installation
+
+```bash
+npm install @password-validator/core
+```
+
+### Basic Usage
+
+```ts
+import { LowerCaseValidator, MaxLengthValidator, MinLengthValidator, PasswordValidatorManager, SpecialCharacterValidator, UpperCaseValidator } from '@password-validator/core';
+
+const pm = new PasswordValidatorManager(); // Create a password validator manager
+
+const minLength = new MinLengthValidator(8); // Minimum length of 8 characters
+const maxLength = new MaxLengthValidator(16); // Maximum length of 16 characters
+const uppercases = new UpperCaseValidator(2); // At least 2 uppercase characters
+const lowercases = new LowerCaseValidator(2); // At least 2 lowercase characters
+const specialCharacters = new SpecialCharacterValidator(2); // At least 2 special characters
+
+pm.register(minLength, maxLength, uppercases, lowercases, specialCharacters);  // Register all the validators with the manager
```

Comment:
Please add a description to inform the users that they need to register the validators before performing password validation with the password validator manager. The description should come above the pm.register();